cpp/src/arrow/flight/sql/odbc/odbc_impl/flight_sql_connection.h

```diff
@@ -28,6 +28,13 @@
 
 namespace arrow::flight::sql::odbc {
 
+/// \brief Case insensitive comparator that takes string_view
+struct CaseInsensitiveComparatorStrView {
```

Comment:
Why is this defined here? It seems to only be used inside the implementation

Reply:
Moved to the .cc file.